libs/components/code-examples/src/lib/modules/core/media-query/responsive-mixins/example.component.scss

```diff
@@ -0,0 +1,21 @@
+@use 'libs/components/theme/src/lib/styles/variables' as vars;
+
+:host {
+  display: block;
+}
+
+.resize-observer-examples {
+  margin: 10px;
+}
+
+.container-queries-examples > .sky-responsive-container-xs {
+  max-width: vars.$sky-screen-xs-max;
```

Comment:
This will throw a lint error under our recommended Stylelint rules because we don't allow the use of any SCSS variables. The only thing the consumer should be using are the mixins in the `@skyux/theme/scss/responsive` entry point.

Reply:
oops, I can remove all this because I went a different route. Forgot about that.